Dockerfile-dev

```diff
@@ -13,8 +13,21 @@ RUN apk add --no-cache build-base yarn postgresql-dev git bash gcompat
 # This should be the same version as the Gemfile.lock
 RUN gem install bundler:2.3.7 --no-document
 
+# Install gems defined in Gemfile
```

Comment:
Did this get put back in on purpose?

Reply:
Yeah; annoyingly it's needed for running the migrations from the command line